xsuite/src/world/lsworld.test.ts

```diff
@@ -766,6 +865,15 @@ test.concurrent("LSWallet.setAccount & LSWallet.getAccount", async () => {
   expect(after).toEqual(before);
 });
 
+test.concurrent("LSWallet.updateAccount", async () => {
+  using world = await LSWorld.start();
+  const { wallet } = await createAccounts(world);
+  const before = await wallet.getAccount();
+  await wallet.updateAccount({ balance: 10n ** 17n });
+  const after = await wallet.getAccount();
+  expect(after).toEqual({ ...before, balance: 10n ** 17n });
```

Comment:
The only thing is that we don't know what "before" is. Let's directly create the wallet without the helper.